test/register_test.js

```diff
@@ -20,9 +20,51 @@ describe('Register tests',  () => {
         agent.close();
     })
 
-    it('Checking phone number length', async () => {
+    it('Checking phone number length not less than 10', async () => {
         const register_res = await agent.post('/register').send({phone: '657488907', username:'stlayer',password:'PaSsW0rd!', name:"Gleep glopb"});
         expect(register_res).to.have.status(400); 
     });
+    it('Checking phone number length not more than 10', async () => {
+        const register_res = await agent.post('/register').send({phone: '65748890721', username:'stlayer',password:'PaSsW0rd!', name:"Gleep glopb"});
+        expect(register_res).to.have.status(400); 
```

Comment:
Please verify the error message along with the status.

Reply:
I changed the phone number in all cases, tests work now

Reply:
I meant the server is returning a message along with the status right ? Verify that as well in `register_res`

Reply:
Added message verification